src/onegov/feriennet/app.py

```diff
@@ -106,12 +111,17 @@ def banner(self, request, id):
         winner = random.choice(candidates)
         winner = winner.compiled(request)
 
+        if winner.name == "ProJuventute":
```

Comment:
A cleaner solution which provides additional functionality would be to make the info also configurable in the yaml (for every language). This would also allow us to use clearer names for the banners (e.g. Projuventute Handy, Projuventute Resilienz, ...)